src/main/java/net/syscon/elite/api/resource/BookingResource.java

```diff
@@ -80,7 +79,7 @@
     @GetMapping
     @ApiOperation(value = "Summary information for all offenders accessible to current user.", notes = "Summary information for all offenders accessible to current user.", nickname = "getOffenderBookings")
     @ApiResponses(value = {
-            @ApiResponse(code = 200, message = "Summary information for all offenders accessible to current user.", response = OffenderBooking.class, responseContainer = "List"),
+            @ApiResponse(code = 200, message = "Summary information for all offenders accessible to current user."),
```

Comment:
spring fox generates the type automatically so better not to specify as it then gets out of sync